storybook/main.js

```diff
@@ -60,11 +60,6 @@ module.exports = {
 		name: '@storybook/react-webpack5',
 		options: {},
 	},
-	features: {
-		babelModeV7: true,
-		emotionAlias: false,
-		storyStoreV7: true,
```

Comment:
Unnecessary, since it's enabled by default since Storybook v7, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storystorev7-enabled-by-default